.conda_env.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - pytorch
   - defaults
 dependencies:
-  - pip=21.2.4
-  - python=3.8.5
+  - python=3.9.16
```

Comment:
Any reason for `3.9.16` specifically? Latest is `3.9.20` I think.

Reply:
The only reason is that I know this `python`+`pip` versions are compatible with each other.